contracts/utils/math/Math.sol

```diff
@@ -114,6 +114,11 @@ library Math {
      * of rounding down.
      */
     function ceilDiv(uint256 a, uint256 b) internal pure returns (uint256) {
+        if (b == 0) {
```

Comment:
This is more than just test improvement. This is a behavior change.

We used to have `ceilDiv(0,0)` return 0, and now it will revert instead. IMO this requires a changeset.

Reply:
You're right, this deserves a changeset. I just added it. Thanks